cmd/opampsupervisor/go.mod

```diff
@@ -34,3 +34,5 @@ require (
 	golang.org/x/sys v0.18.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace go.opentelemetry.io/collector/confmap => github.com/atoulme/opentelemetry-collector/confmap v0.0.0-20240503145806-395864958aba
```

Comment:
Usage of `atoulme` fork is just temporary?

Reply:
yes, just to prove tests pass with the core changes.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/9862#issuecomment-2100320626 see context here.